packages/next/lib/typescript/writeConfigurationDefaults.ts

```diff
@@ -193,13 +193,6 @@ export async function writeConfigurationDefaults(
     return
   }
 
-  console.log(
-    chalk.green(
-      `We detected TypeScript in your project and reconfigured your ${chalk.bold(
-        'tsconfig.json'
-      )} file for you. Strict-mode is set to ${chalk.bold('false')} by default.`
```

Comment:
Removing the message is not the way to go. It should either detect if strict mode wasn't set previously and not show that part of the message or the message can be expanded to include "if it was not set previously".

Reply:
I'm assuming you've misinterpreted the comment above given that the message is still removed. There's 2 options in my comment above:

- Detect if strict mode wasn't set previously and don't show the strict mode message
- Change the message to `Strict-mode is set to ${chalk.bold('false')} by default if it was not set previously`

Personally prefer the first option given that it would have the result you want only when strict mode was set already 👍